dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.34/V2_34_3__Update_program_rule_actions_and_update_evaluation_time.sql

```diff
@@ -0,0 +1,3 @@
+update programruleaction set actiontype = 'SHOWMESSAGE' where actiontype = 'DISPLAYKEYVALUEPAIR' OR actiontype = 'DISPLAYTEXT';
+update programruleaction set actiontype = 'SHOWWARNING', evaluationtime = 'ON_COMPLETE' where actiontype = 'WARNINGONCOMPLETE';
+update programruleaction set actiontype = 'SHOWERROR', evaluationtime = 'ON_COMPLETE' where actiontype = 'ERRORONCOMPLETE';
```

Comment:
Assuming we must also set the WHEN parameter properly for these(on complete)

Reply:
I am doing it.
I am changing the actionType and the evaluationTime